test/test_dockerfile_hadolint_linter.vader

```diff
@@ -55,15 +56,30 @@ Execute(command is correct when using docker):
   let b:ale_dockerfile_hadolint_use_docker = 'always'
 
   AssertEqual
-  \ "docker run --rm -i hadolint/hadolint hadolint --no-color -",
+  \ "docker run --rm -i hadolint/hadolint hadolint  --no-color -",
```

Comment:
This isn't really my favorite change to need to make, but this was necessary for a one-line change in the actual source if I wasn't going to make a giant `if else` to see if arguments were provided and we needed an extra space or not